web/beacon-app/src/features/home/components/QuickStart.tsx

```diff
@@ -1,12 +1,9 @@
-import OtterLookingDown from '@/components/icons/otter-looking-down';
-
 import SetupNewProject from '../components/SetupNewProject';
 import AccessDocumentationStep from './AccessDocumentationStep';
 
 export default function QuickStart() {
   return (
-    <div className="relative w-full space-y-10">
-      <OtterLookingDown className="hidden md:absolute md:-top-[215px] md:-right-20 md:block md:scale-[.65] " />
+    <div className="w-full space-y-10 pt-10">
```

Comment:
`pt-10` was added above the `Get Started`. Since that was deleted, the padding was added here for now. It may change as the home page refactor continues.